datasets/gov/vat/r34_isduoti_aap.csv

```diff
@@ -0,0 +1,74 @@
+,,,,,id,string,,,,4,open,,Identifikacinis numeris,
+,,,,,produktas,string,,,,4,open,,AAP pavadinimas,Bendroji informacija 
+,,,,,busena,string,,,,4,open,,Registracijos būsena,
+,,,,,,enum,,,"""REGISTRUOTAS_120_D""",,,,,
+,,,,,,,,,"""TAPATUS""",,,,,
+,,,,,savininkas,string,,,,1,open,,AAP registracijos savininkas,Bendroji informacija 
+,,,,,atstovas,string,,,,1,open,,AAP registracijos savininko atstovas,Bendroji informacija 
+,,,,,registracijos_nr,string,,,,4,open,,Registravimo Nr. Lietuvoje,Registracijos duomenys 
+,,,,,registracijos_atnaujinimas,date,D,,,4,open,,Registracijos atnaujinimo data,
+,,,,,registracijos_pradzia,date,D,,,4,open,,AAP registracijos data,Registracijos duomenys 
+,,,,,registracijos_pabaiga,date,D,,,4,open,,AAP registracijos pabaigos data,Registracijos duomenys 
+,,,,,naudoti_iki,date,D,,,4,open,,Pasibaigus registracijai naudoti iki,Registracijos duomenys  
+,,,,,prekiauti_iki,date,D,,,4,open,,Pasibaigus registracijai prekiauti iki,Registracijos duomenys  
+,,,,,medziagos_aprasymas,string,,,,2,open,,Cheminė medžiaga / Mikroorganizmas,Cheminės medžiagos / Mikroorganizmo suformuotas aprašymas
+,,,,,veiklioji_medziaga,string,,,,4,open,,Cheminė medžiaga / Mikroorganizmas,
+,,,,,,enum,,,"""Cheminė medžiaga""",,,,,
+,,,,,,,,,"""Mikroorganizmas""",,,,,
+,,,,,veikimo_grupe,string,,,,3,open,,Veikimo grupė,Veikimo grupių suformuotas aprašymas
+,,,,,aap_tipas,string,,,,4,open,,Augalų apsaugos produkto tipas,
+,,,,,,enum,,,"""CHEMINIS""",,,,,
+,,,,,,,,,"""MIKROBIOLOGINIS""",,,,,
+,,,,,augalas,string,,,,3,open,,Lietuviškas augalo pavadinimas,Numatomi naudojimai (GAP) 
+,,,,,kenksmingas_organizmas,string,,,,2,open,,Lietuviški kenksmingųjų organizmų pavadinimai,Numatomi naudojimai (GAP) 
+,,,,,numatomas_naudojimas,string,,,,4,open,,Naudotojas,Numatomi naudojimai (GAP) 
+,,,,,,enum,,,"""PROFESIONALIAJAM_NAUDOJIMUI""",,,,,
+,,,,,,,,,"""NEPROFESIONALIAJAM_NAUDOJIMUI""",,,,,
+,,,,,naudojimo_budas,string,,,,2,open,,Naudojimo būdas,Naudojimo būdų suformuotas aprašymas
+,,,,,naudojimo_plotas,string,,,,4,open,,Maži / dideli plotai,Numatomi naudojimai (GAP)
+,,,,,,enum,,,"""DIDELIAIS_PLOTAIS""",,,,,
+,,,,,,,,,"""MAZAIS_PLOTAIS""",,,,,
+,,,,,produkto_forma,string,,,,3,open,,Produkto forma,Bendroji informacija 
+,,,,,platintojas,string,,,,1,open,,Platinojas,Bendroji informacija 
+,,,,,pakuote,string,,,,1,open,,Produkto pakuotės,Bendroji informacija 
+,,,,,pastaba,string,,,,2,open,,Pastabos ,Bendroji informacija 
+,,,,,medz_cas_nr,string,,,,4,open,,Medžiagos CAS Nr. / Mikroorganizmo Nr.,Veiklioji medžiaga 
```

Comment:
Kokie tai duomenys? Palantyre nėra nė vieno įrašo.
Ir ar šie duomenys nesisieja su kuriuo nors VAT rinkiniu? Panašius pavadinimus lyg esu mačiusi kituose VAT rinkiniuose.

Reply:
Ačiū už pastebėjimą dėl `apsaug_medz_cas`. Šis stulpelis yra visiškai tuščias, tad jo neeksportuosiu (išimsiu jį iš lentos, kad nemaišytų). Šiaip visas šis rinkinys ateina iš tos pačios lentos kaip ir R33, tik čia taikomi skirtingi filtrai. Ref'ų su kitais atvertais rinkiniais šiuo atveju nematyčiau.